var/spack/repos/builtin/packages/rocprim/package.py

```diff
@@ -149,8 +150,8 @@ def setup_build_environment(self, env):
 
     def cmake_args(self):
         args = [
-            self.define("ONLY_INSTALL", "ON"),
-            self.define("BUILD_TEST", "OFF"),
+            self.define("ONLY_INSTALL", (not self.run_tests)),
+            self.define("BUILD_TEST", self.run_tests),
```

Comment:
Are these two options one the opposite of the other? :thinking:

Reply:
Yes, the tests will build when ONLY_INSTALL is off and BUILD_TEST is on